index.md

```diff
@@ -39,21 +39,14 @@ Here are a few values we think you will receive from participating in this proje
 
 ## Schedule
 
-The first phase of Open Life Science will launch in early 2020!
```

Comment:
I would keep the important date here on the index page (that they got the information directly) and repeat them in the schedule page later

Reply:
Agree. The date will also keep changing (if we continue).